Dispatcher/Resources/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,74 @@
+{
+  "images" : [
+    {
+      "filename" : "AppStore-1024@1x 1.png",
```

Comment:
give the images meaningful names
what is appStore? what is iPad?
theres no way for other developers to understand where those icons belong